app/base/models.py

```diff
@@ -23,20 +23,30 @@ class User(db.Model, UserMixin):
     def json(self):
         return {
             "id": self.id,
-            "username": self.name,
+            "username": self.username,
             "email": self.email,
             "profile_image": self.profile_image,
         }
 
     @staticmethod
     def get_all_users():
-        return [User.json(user) for user in User.query.all]
+        return [User.json(user) for user in User.query.all()]
 
     @staticmethod
     def get_user(_user_id):
         query = User.query.get(_user_id)
         return query
 
+    @staticmethod
+    def username_exists(_username):
+        return_value = User.query.filter_by(username=_username).first()
+        return bool(return_value)
+
+    @staticmethod
+    def email_exists(_email):
+        return_value = User.query.filter_by(email=_email).first()
+        return bool(return_value)
```

Comment:
similar here 

```
    @staticmethod
    def email_exists(_email):
        return bool(User.query.filter_by(email=_email).first())
```

Reply:
Same for here. I couldn't get it to work. I got AttributeError: 'NoneType' object has no attribute 'exist'

Reply:
same here,
that was my typo